registration-processor-default.properties

```diff
@@ -989,6 +990,8 @@ landing.zone.account.name=landing-zone
 landing.zone.type=DMZServer
```

Comment:
Please update these properties in this line only,dont add new

Reply:
Done